speed-dreams/source-2.2.3/src/libs/robottools/rthumandriver.cpp

```diff
@@ -616,8 +616,9 @@ void HumanDriver::new_race(int index, tCarElt* car, tSituation *s)
         tIndicator indicatorSettings = mediator->GetIndicatorSettings();
         InterventionType interventionType = mediator->GetInterventionType();
         tParticipantControl participantControlSettings = mediator->GetPControlSettings();
+        ReplayOption replayRecorderOption = mediator->GetReplayRecorderSetting();
```

Comment:
You use ReplayOption here but bool in other places, personally I don't think a typedef is needed and bool is fine.